### PR TITLE
Adds a troubleshooting note about powershell versions <3.

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

**Comment by zack-shoylev** [comment]
@everett-toews troubleshooting note as requested.

**Comment by everett-toews** [comment]
Can you please move this to https://github.com/rackerlabs/docs-container-service/blob/master/_tutorials/070-troubleshooting.md instead as it could happen anytime a user tries to run `docker.ps1`?

Include text from the actual error message too and that's the kind of thing devs tend to Google for. Thanks.

**Comment by zack-shoylev** [comment]
Good points, will do.

**Comment by zack-shoylev** [comment]
@everett-toews updated PR.

File: _tutorials/070-troubleshooting.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ $ docker exec test echo "Yay! :)" || echo "Aw Man :("
 Yay! :)
 ```
 
+### Old PowerShell version on Windows
+
+If you experience this issue, you will see:
```
Comment (everett-toews) [inline _tutorials/070-troubleshooting.md]: If you are running PowerShell version 2 or less, the $PSScriptRoot variable in docker.ps1 is not supported and you get the following error message:

File: _tutorials/070-troubleshooting.md [hunk 2]
```diff
@@ -78,6 +78,18 @@ $ docker exec test echo "Yay! :)" || echo "Aw Man :("
...
+
+```
+PS C:\Users\username\Desktop\mycluster> docker info
+Could not read CA certificate "C:\\Users\\username\\.docker\\ca.pem": open C:\Users\username\.docker\ca.pem: The system cannot find the file specified.
+```
+
+If you do not have PowerShell version 3 or above, the $PSScriptRoot variable in docker.ps1 is not supported, and the environment is not set up properly.
```
Comment (everett-toews) [inline _tutorials/070-troubleshooting.md]: Delete this sentence.

File: _tutorials/070-troubleshooting.md [hunk 3]
```diff
@@ -78,6 +78,18 @@ $ docker exec test echo "Yay! :)" || echo "Aw Man :("
...
+Upgrade PowerShell to version 3 or above to fix this issue.
```
Comment (everett-toews) [inline _tutorials/070-troubleshooting.md]: To resolve this error, upgrade PowerShell to version 3 or above.

**Comment by zack-shoylev** [comment]
Thanks @everett-toews

**Comment by everett-toews** [comment]
Thanks @zack-shoylev!

**Pull request merged**